lib/ioki/model/operator/line_stop.rb

```diff
@@ -4,19 +4,65 @@ module Ioki
   module Model
     module Operator
       class LineStop < Base
-        attribute :type, on: :read, type: :string
-        attribute :id, on: :read, type: :string
-        attribute :created_at, on: :read, type: :date_time
-        attribute :updated_at, on: :read, type: :date_time
-        attribute :tier, on: [:read, :create, :update], type: :integer
-        attribute :relative_time, on: [:read, :create, :update], type: :integer
-        attribute :dropoff_mode, on: [:read, :create, :update], type: :string, omit_if_nil_on: [:create, :update]
-        attribute :pickup_mode, on: [:read, :create, :update], type: :string, omit_if_nil_on: [:create, :update]
-        attribute :supports_pickup, on: [:read, :create, :update], type: :boolean
-        attribute :supports_dropoff, on: [:read, :create, :update], type: :boolean
-        attribute :supports_pass_through, on: [:read, :create, :update], type: :boolean
-        attribute :station_id, on: [:read, :create], type: :string
-        attribute :version, on: :read, type: :integer
+        attribute :type,
+                  on:   :read,
+                  type: :string
+
+        attribute :id,
+                  on:   :read,
+                  type: :string
+
+        attribute :created_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :updated_at,
+                  on:   :read,
+                  type: :date_time
+
+        attribute :tier,
+                  on:             [:read, :create, :update],
+                  omit_if_nil_on: [:create, :update],
+                  type:           :integer
+
+        attribute :relative_time,
+                  on:             [:read, :create, :update],
+                  omit_if_nil_on: [:create, :update],
```

Comment:
`relative_time` is required on `create`.

Reply:
sounds reasonable to me. done 👍